enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,179 @@
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes a mechanism for blocking edges for the subset of clusters considered vulnerable to known issues with the stability of a particular update or target release.
+
+## Motivation
+
+When managing the [Cincinnati][cincinnati-spec] update graph [for OpenShift][cincinnati-for-openshift-design], we sometimes discover stability issues with particular release images or updates between them.
+Once an issue is discovered, we [block edges][block-edges] so we no longer recommend risky updates, or updates to risky releases.
+
+Note: as described [in the documentation][support-documentation], supported updates are still supported even if incoming edges are blocked, and Red Hat will eventually provide supported update paths from any supported release to the latest supported release in its z-stream.
+
+Incoming bugs are evaluated to determine an impact statement based on [a generic template][rhbz-1858026-impact-statement-request].
+Some bugs only impact specific platforms, or clusters with other specific features.
+For example, rhbz#1858026 [only impacted][rhbz-1858026-impact-statement] clusters with the `None` platform which were created as 4.1 clusters and subsequently updated via 4.2, 4.3, and 4.4 to reach 4.5.
+In those cases there is currently tension between wanting to protect vulnerable clusters by blocking the edge vs. wanting to avoid inconveniencing clusters which we know are not vulnerable and whose administrators may have been planning on taking the recommended update.
+
+### Goals
+
+* [Cincinnati graph-data][graph-data] maintainers will have the ability to block edges for the subset of clusters matching a particular Prometheus query.
+
+### Non-Goals
+
+* Blocking edges based on data that is not [uploaded to Telemetry][uploaded-telemetry].
+* Blocking edges for subsets of clusters when the update service is unable to reach or authenticate with an aggregating Prometheus server.
+    For example, users running a local update service will not be able to access Red Hat's internal Telemetry to determine the requesting cluster's platform, etc.
+* Specifying a particular update service implementation.
+    This enhancement floats some ideas, but the details of the chosen approach are up to each update service's maintainers.
+
+## Proposal
+
+### Enhanced graph-data schema for blocking edges
+
+[The blocked-edges schema][block-edges] will be extended with the following new properties:
+
+* `clusters` (optional, [object][json-object]), defining the subset of affected clusters.
+    If any `clusters` property matches a given cluster, the edge should be blocked for that cluster.
+    * `prometheus` (optional, [object][json-string]), with a [PromQL][] query describing affected clusters.
```

Comment:
```suggestion
    * `telemetry` (optional, [object][json-string]), with a [PromQL][] query describing affected clusters.
```
That would highlight the source of the data (we might want to add `ccx` source later if data from Insights operator would be required)

Reply:
I flipped it to `prometheus`, because Cincy doesn't care about the source, as long as it processes PromQL.  We will point our deploy at Telemetry.  Folks running Cincy locally may point it at their own Prom aggregator.

Reply:
Sounds good to me, perhaps its worth renaming to `promql` for clarity (but I'm not insisting)

Reply:
Pivoted to `promql` with d2f0889 -> 1f64222.